config/config.yaml

```diff
@@ -53,3 +53,8 @@ tls:
     detector_bundle_no_ca:
         cert_path: /path/to/client-bundle.pem
         insecure: true
+
+# Following section can be used to configure the allowed headers that orchestrator will pass to
+# NLP provider and detectors. Note that, this section takes header keys, not values.
+# allowed_headers_passthrough:
+#     - "header-key"
```

Comment:
Should this value be quoted? It is not quoted on the [test](https://github.com/foundation-model-stack/fms-guardrails-orchestrator/pull/214/files#diff-cba64c21ab992eaad29fce147a08f4560a4769bc14682b8a96081a5fd02dbecdR601).

Reply:
it doesn't need to be.